tests/Aspire.Dashboard.Tests/GlobalizationTests.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace Aspire.Dashboard.Tests;
+
+public class GlobalizationTests
+{
+    [Fact]
+    public void GetSupportedCultures_IncludesPopularCultures()
```

Comment:
nit: Add arrange, act, assert comments to tests. See other tests for examples. They're a simple way to add a bit of structure to tests. Not that valuable in short tests like this, but very useful in longer tests.

Reply:
I don't see any arrange part, but I added act and assert comments